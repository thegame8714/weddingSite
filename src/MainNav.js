import React from 'react'
import { Link } from 'react-router-dom'

const MainNav = () => (
  <ul className="main-nav">
    <li>
      <Link to="/">Home</Link>
    </li>
    <li>
      <Link to="/event">Event</Link>
    </li>
    {
      //<li>Confirm Presence</li>
      //<li>Dietary Requirements</li>
    }
  </ul>
)

export default MainNav
