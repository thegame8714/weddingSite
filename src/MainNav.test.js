import React from 'react'
import renderer from 'react-test-renderer'
import MainNav from './MainNav'
// import { Router } from 'react-router-dom'

describe.skip('MainNav component', () => {
  it('should show render the main menu', () => {
    const tree = renderer.create(<MainNav />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
