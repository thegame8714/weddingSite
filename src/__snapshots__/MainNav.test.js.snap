// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNav component should show render the main menu 1`] = `
<ul>
  <li>
    Home
  </li>
  <li>
    Event
  </li>
  <li>
    Confirm Presence
  </li>
  <li>
    Dietary Requirements
  </li>
</ul>
`;
